app/lib/pre_assembly/digital_object.rb

```diff
@@ -9,7 +9,7 @@ class DigitalObject
                 :stager, :label, :druid, :source_id, :container
 
     delegate :staging_location,
-             :content_md_creation,
+             :processing_configuration,
              :content_structure,
```

Comment:
note: `content_md_creation` is now `processing_configuration` in `Batch` as noted above